src/main/java/org/terasology/ligthandshadow/componentsystem/LASUtils.java

```diff
@@ -40,7 +40,7 @@
      * Base is a square of side 2 * BASE_EXTENT + 1 with the flag at the center
      */
     public static final int BASE_EXTENT = 2;
-    public static final int GOAL_SCORE = 5;
+    public static final int GOAL_SCORE = 2;
```

Comment:
Was this values lowered for testing? I feel like it shouln't be changed for the final merge without reason.